modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/CustomizedTableFDSView.java

```diff
@@ -155,6 +157,24 @@ public boolean isQuickActionsEnabled() {
 		return true;
 	}
 
+	private DateTimeFDSTableSchemaField _addDateFDSTableSchemaField() {
+		DateTimeFDSTableSchemaField dateFDSTableSchemaField =
+			new DateTimeFDSTableSchemaField();
+
+		dateFDSTableSchemaField.setContentRenderer(
+			"dateTime"
+		).setFieldName(
+			"date"
+		).setLabel(
+			"date"
+		);
+
+		dateFDSTableSchemaField.setTimeZoneBehavior(
+			FDSTimeZoneBehaviors.APPLY_THEME_DISPLAY_TIME_ZONE);
+
+		return dateFDSTableSchemaField;
+	}
```

Comment:
In difference to previous version, I set the usage example in our sample module, so we can safely push the solution. Commerce team can apply it to their module in a followup.

Reply:
this keeps the commerce tables just as they behave at this moment. We'd like to let commerce people know so that the make the adjustments later on.

For context for @smottal, origin of this issue is a customer who wanted to see the dates in the user-profile timezone. Rather than hardcoding the behavior in the FDS we're making it condfigurable by code. So now it's up to the different use cases to tell FDS how they want to render dates.